spigot/src/main/java/com/github/camotoy/geyserskinmanager/spigot/listener/SpigotPlatformEventListener.java

```diff
@@ -82,7 +88,45 @@ protected void uploadOrRetrieveSkin(MinecraftProfileWrapper profile, Player play
             }
         }
     }
-
+    
+    public String[] RetrieveSkin(Player player) {
+
+        PlayerProfile playerProfile = player.getPlayerProfile();
```

Comment:
This cannot be here - it's a Paper-only property, and thus will fail on Spigot. Look at my `MinecraftProfileWrapper` class for an idea of how to work around this.